src/cloudwatch/modules/configuration/metadatareader.py

```diff
@@ -59,13 +62,31 @@ def _get_metadata(self, request):
                    'http://169.254.169.254/latest/meta-data/placement/availability-zone/' 
                    then the request part is 'latest/meta-data/placement/availability-zone/'.
         """
-        result = self.session.get(self.metadata_server + request, timeout=self._REQUEST_TIMEOUT)
+        headers = {'X-aws-ec2-metadata-token':self.token}
```

Comment:
better define a constant for the header

Reply:
self.token will vary every time. it can not be defined as a constant variable.
Yes.  I can define 'X-aws-ec2-metadata-token' as a constant string.